extensions/amp-video-docking/0.1/amp-video-docking.js

```diff
@@ -271,7 +271,7 @@ export class VideoDocking {
      * Overriden when user drags the video to a different corner.
      * @private {!DirectionX}
      */
-    this.cornerDirectionX_ = isRTL(this.getDoc_())
+    this.cornerDirectionX_ = !!this.getDoc_().body && isRTL(this.getDoc_())
```

Comment:
This would ignore cases when the body is not ready but the document is RTL.

Let's set it to `null` instead. `getUsableTarget_()` will always execute on events that are sent after the body is ready, so we should resolve it there before returning:

```js
    if (this.cornerDirectionX_ === null) {
      this.cornerDirectionX_ = isRTL(this.getDoc_())
        ? DirectionX.LEFT
        : DirectionX.RIGHT;
    }

    return {
      type: DockTargetType.CORNER,
      ...
    };
```

If you have issues with the tests or this is otherwise more involved, let me know and I can take this off your hands :)